### DISCUSSION TITLE
How do I change the language?

### DISCUSSION THREAD

**Discussion by NickSw1ft** [flxzt/rnote#698] [Q&A]
I want to change the app language from my system's default to English. The problem is that there is no such setting in "settings"
I am using Windows 10 btw

**Comment by flxzt** [comment]
It is not currently possible to change the language from within the app, it always follows the system language.

**Comment by NickSw1ft** [comment]
Can I change it somehow in configs file?

**Reply by eliblaster** [reply]
delete corresponding language folder in the C:\...\Rnote\share\locale folder
